styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -547,19 +550,30 @@ private void setupMetrics(
                   .count()));
       stats.registerActiveStates(
           state,
-          "none",
-          () -> stateManager.activeStates().values().stream()
+          "none", () -> stateManager.activeStates().values().stream()
               .filter(runState -> runState.state().equals(state))
               .filter(runState -> !runState.data().trigger().isPresent())
               .count());
     });
 
-    stats.registerQueuedEvents(queuedEventsCount);
-    stats.registerWorkflowCount("all", allWorkflowsCount);
-    stats.registerWorkflowCount("configured", configuredWorkflowsCount);
-    stats.registerWorkflowCount("enabled", configuredEnabledWorkflowsCount);
-    stats.registerWorkflowCount("docker_termination_logging_enabled",
-                                dockerTerminationLoggingEnabledWorkflowsCount);
+    final Function<String, Gauge<Long>> createResourceCountGauge = (resource) -> {
```

Comment:
Shouldn't call it "resource count" if it's "resource usage" everywhere else. It even has a different meaning.